otx/cli/tools/train.py

```diff
@@ -250,8 +257,23 @@ def main():  # pylint: disable=too-many-branches
             prediction_dataset=predicted_validation_dataset,
         )
         task.evaluate(resultset)
-        assert resultset.performance is not None
-        print(resultset.performance)
+        performance = resultset.performance
+        assert performance is not None
+        print(performance)
+
+    end_time = time.time()
+    sec = end_time - start_time
+    total_time = str(datetime.timedelta(seconds=sec))
+    print("otx train time elapsed: ", total_time)
+    model_results = {"time elapsed": total_time, "score": performance, "model_path": str(model_path.absolute())}
```

Comment:
Hmm, it's just a question, do you have any plan to include `trained epochs` and `best epoch` information on other PR?

Reply:
> I don't know what trained epoch means, but I don't plan on doing that right now.

As you can see from the code configuration, it is possible by including that information in task.config.